romanisim/tests/test_l1.py

```diff
@@ -84,6 +84,33 @@ def test_apportion_counts_to_resultants():
     log.info('DMS229: successfully generated ramp from counts.')
 
 
+def test_linearized_counts_to_resultants():
+    """Test that we can apportion linearized counts to resultants.
+    """
+    size = (4,4)
+    counts = np.random.poisson(8, size=size)
+
+    coeffs = np.asfarray([0, 0.5, 0, 0])
+    lin_coeffs = np.tile(coeffs[:,np.newaxis,np.newaxis], (1, 4, 4))
+
+    rng = galsim.UniformDeviate(42)
+
+    inv_linearity = nonlinearity.NL(lin_coeffs)
+
+    for tij in tijlist:
+        resultants, dq = l1.apportion_counts_to_resultants(counts.copy(), tij.copy(), rng=rng)
+
+        res2, dq2 = l1.apportion_counts_to_resultants(counts.copy(), tij.copy(), rng=rng,
+                                                      inv_linearity=inv_linearity)
+
+        # Ensure that the linear coefficients were actually applied
+        assert np.all(res2 >= 0)
+        assert np.any(res2.sum(axis=0) < resultants.sum(axis=0))
+        assert np.all(res2.sum(axis=0) <= resultants.sum(axis=0))
```

Comment:
Can we require something like the median of the difference between the first and last resultant changes by something roughly like a factor of two?  Might need to make this more like 100x100 than 4x4 for that to come out decently closely.

Reply:
Done, and it matches well.